librz/core/cmd_analysis.c

```diff
@@ -857,16 +857,14 @@ static bool cmd_analysis_aaft(RzCore *core) {
 	RzListIter *it;
 	RzAnalysisFunction *fcn;
 	ut64 seek;
-	const char *io_cache_key = "io.pcache.write";
-	bool io_cache = rz_config_get_i(core->config, io_cache_key);
+	RzConfigHold *hold = rz_config_hold_new(core->config);
 	if (rz_config_get_i(core->config, "cfg.debug")) {
 		eprintf("TOFIX: aaft can't run in debugger mode.\n");
 		return false;
 	}
-	if (!io_cache) {
-		// XXX. we shouldnt need this, but it breaks 'rizin -c aaa -w ls'
-		rz_config_set_i(core->config, io_cache_key, true);
-	}
+	rz_config_hold_i(hold, "io.va", "io.pcache.write", NULL);
+	// XXX. we shouldnt need this, but it breaks 'rizin -c aaa -w ls'
+	rz_config_set_i(core->config, "io.pcache.write", true);
```

Comment:
I'm not sure I understand the initial logic, but this was under a condition `if (!io_cache)`. Is it alright to avoid the condition?